src/shared/utilities/childProcess.ts

```diff
@@ -8,6 +8,13 @@
 import * as child_process from 'child_process'
 import * as crossSpawn from 'cross-spawn'
 
+export interface ChildProcessStartArguments {
```

Comment:
Out of scope for this change, but I wonder if we should rename `ChildProcess` and its associated types to avoid confusion with `child_process.ChildProcess` and its associated types.